spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -596,15 +595,33 @@ function sendPacket(
     // increment the sequence. Thus there are monotonically increasing sequences for packet flow for a given clientId
     nextSequenceSend[sourceChannelId]=sequence+1
     
-    // log that a packet can be safely sent
-    // Event Emission 
-    emitLogEntry("sendPacket", {
-      sourceId: sourceChannelId, 
-      destId: channel.counterpartyChannelId, 
-      sequence: sequence,
-      packet: packet,
-      timeoutTimestamp: timeoutTimestamp, 
+    // Event Emission for send packet
+    emitEvents("send_packet", {
+      sourceChannel: sourceChannelId, 
+      destChannel: channel.counterpartyChannelId,
+      sequence: sequence, // value is string in decimal format
+      timeoutTimestamp: timeoutTimestamp, // value is string in decimal format
+      payloadLength: len(payloads), // value is string in decimal format
+      // include first payload data in events if there is only one payload
+      version: payload[0].version,
+      encoding: payload[0].encoding,
+      data: toHex(payload[0].appData) // emit app bytes as string in hex format
     })
+
+    // for multi payload cases, we will emit each payload as a separate event
+    // these will include the packet identifier so they can be indexed and
+    // reconstructed by relayers
+    for i, payload in payloads {
```

Comment:
In the single payload case, a `send_payload` event would be emitted anyway. Since we emit an event with the first payload related data in the `send_packet` event, this results in a duplicated field emission. 
Do we want to emit the `send_payload` event even in the single payload case? 
If not, shall we discard the first element of the payload array here? Alternatively, shall we remove version,encoding,data from the `send_packet` event?